DEBUG - 2025-07-11 05:59:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 06:00:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 06:00:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 06:01:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 06:01:55 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-07-11 06:40:49 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 06:40:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 06:40:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:40:49\', \'2025-07-11 06:40:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:40:49\', \'2025-07-11 06:40:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 06:40:49 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:40:49\', \'2025-07-11 06:40:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:40:49\', \'2025-07-11 06:40:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:40:49\', \'2025-07-11 06:40:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 06:51:11 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 06:51:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 06:51:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:51:11\', \'2025-07-11 06:51:11\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:51:11\', \'2025-07-11 06:51:11\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 06:51:11 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:51:11\', \'2025-07-11 06:51:11\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:51:11\', \'2025-07-11 06:51:11\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'1111111111111111\', \'2025-07-11 06:51:11\', \'2025-07-11 06:51:11\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 07:00:16 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 07:00:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:00:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'dwfsdff\', \'sdfsfsf\', \'sdfsadfsa\', \'sdfsfsdf\', \'2025-07-11 07:00:16\', \'2025-07-11 07:00:16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'dwfsdff\', \'sdfsfsf\', \'sdfsadfsa\', \'sdfsfsdf\', \'2025-07-11 07:00:16\', \'2025-07-11 07:00:16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:00:16 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'dwfsdff\', \'sdfsfsf\', \'sdfsadfsa\', \'sdfsfsdf\', \'2025-07-11 07:00:16\', \'2025-07-11 07:00:16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'dwfsdff\', \'sdfsfsf\', \'sdfsadfsa\', \'sdfsfsdf\', \'2025-07-11 07:00:16\', \'2025-07-11 07:00:16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'dwfsdff\', \'sdfsfsf\', \'sdfsadfsa\', \'sdfsfsdf\', \'2025-07-11 07:00:16\', \'2025-07-11 07:00:16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 07:01:49 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 07:01:49 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:01:49 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:01:49\', \'2025-07-11 07:01:49\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:01:49\', \'2025-07-11 07:01:49\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:01:49 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:01:49\', \'2025-07-11 07:01:49\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:01:49\', \'2025-07-11 07:01:49\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:01:49\', \'2025-07-11 07:01:49\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 07:03:16 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(94): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 07:03:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:03:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:03:16\', \'2025-07-11 07:03:16\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:03:16\', \'2025-07-11 07:03:16\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:03:16 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:03:16\', \'2025-07-11 07:03:16\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:03:16\', \'2025-07-11 07:03:16\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (\'jhhjkhkj\', \'jkhljkhkj\', \'kjhljkhjk\', \'kjhjlkhljk\', \'1111111111111111\', \'2025-07-11 07:03:16\', \'2025-07-11 07:03:16\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(94): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-07-11 07:42:58 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 07:44:38 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-07-11 07:47:14 --> ErrorException: Undefined variable $emp_id
[Method: GET, Route: new_emp_joining]
in APPPATH\Views\rms\rms_new_joining.php on line 539.
 1 APPPATH\Views\rms\rms_new_joining.php(539): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $emp_id', 'C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php', 539)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('rms/rms_new_joining', [], true)
 5 APPPATH\Controllers\RmsController.php(16): view('rms/rms_new_joining', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->new_joining()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:49:56 --> ErrorException: Undefined variable $emp_id
[Method: GET, Route: new_emp_joining]
in APPPATH\Views\rms\rms_new_joining.php on line 539.
 1 APPPATH\Views\rms\rms_new_joining.php(539): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $emp_id', 'C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php', 539)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('rms/rms_new_joining', [], true)
 5 APPPATH\Controllers\RmsController.php(16): view('rms/rms_new_joining', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->new_joining()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 07:53:36 --> ErrorException: Undefined variable $emp_id
[Method: GET, Route: new_emp_joining]
in APPPATH\Views\rms\rms_new_joining.php on line 539.
 1 APPPATH\Views\rms\rms_new_joining.php(539): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $emp_id', 'C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php', 539)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('rms/rms_new_joining', [], true)
 5 APPPATH\Controllers\RmsController.php(16): view('rms/rms_new_joining', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->new_joining()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-07-11 07:56:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-07-11 07:56:21 --> ErrorException: Undefined variable $emp_id
[Method: GET, Route: new_emp_joining]
in APPPATH\Views\rms\rms_new_joining.php on line 539.
 1 APPPATH\Views\rms\rms_new_joining.php(539): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $emp_id', 'C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php', 539)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('rms/rms_new_joining', [], true)
 5 APPPATH\Controllers\RmsController.php(16): view('rms/rms_new_joining', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->new_joining()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-07-11 08:36:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
CRITICAL - 2025-07-11 08:36:11 --> ErrorException: Undefined variable $emp_id
[Method: GET, Route: new_emp_joining]
in APPPATH\Views\rms\rms_new_joining.php on line 539.
 1 APPPATH\Views\rms\rms_new_joining.php(539): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $emp_id', 'C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php', 539)
 2 SYSTEMPATH\View\View.php(224): include('C:\\xampp\\htdocs\\tempp\\app\\Views\\rms\\rms_new_joining.php')
 3 SYSTEMPATH\View\View.php(227): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1173): CodeIgniter\View\View->render('rms/rms_new_joining', [], true)
 5 APPPATH\Controllers\RmsController.php(16): view('rms/rms_new_joining', [...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->new_joining()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-07-11 08:56:41 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 08:59:40 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2025-07-11 09:37:48 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(130): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 09:37:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:37:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:37:48\', \'2025-07-11 09:37:48\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:37:48\', \'2025-07-11 09:37:48\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:37:48 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:37:48\', \'2025-07-11 09:37:48\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:37:48\', \'2025-07-11 09:37:48\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:37:48\', \'2025-07-11 09:37:48\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 09:38:09 --> mysqli_sql_exception: Unknown column 'created_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(130): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 09:38:09 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:38:09 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:38:09\', \'2025-07-11 09:38:09\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:38:09\', \'2025-07-11 09:38:09\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:38:09 --> [Caused by] mysqli_sql_exception: Unknown column 'created_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:38:09\', \'2025-07-11 09:38:09\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:38:09\', \'2025-07-11 09:38:09\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:38:09\', \'2025-07-11 09:38:09\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2025-07-11 09:39:48 --> mysqli_sql_exception: Unknown column 'updated_at' in 'field list' in C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php:327
Stack trace:
#0 C:\xampp\htdocs\tempp\system\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `id...', 0)
#1 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `id...')
#2 C:\xampp\htdocs\tempp\system\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `id...')
#3 C:\xampp\htdocs\tempp\system\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `id...', Array, false)
#4 C:\xampp\htdocs\tempp\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\tempp\system\BaseModel.php(843): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\tempp\system\Model.php(800): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\tempp\app\Controllers\RmsController.php(130): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\tempp\system\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
#9 C:\xampp\htdocs\tempp\system\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
#10 C:\xampp\htdocs\tempp\system\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\tempp\system\Boot.php(334): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\tempp\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\tempp\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2025-07-11 09:39:48 --> CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
[Method: POST, Route: identification_details_save]
in SYSTEMPATH\Database\BaseConnection.php on line 692.
 1 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 7 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
 8 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:39:48 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 332.
 1 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:39:48\', \'2025-07-11 09:39:48\')')
 2 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:39:48\', \'2025-07-11 09:39:48\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
 9 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
10 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2025-07-11 09:39:48 --> [Caused by] mysqli_sql_exception: Unknown column 'updated_at' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 327.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(327): mysqli->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:39:48\', \'2025-07-11 09:39:48\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:39:48\', \'2025-07-11 09:39:48\')')
 3 SYSTEMPATH\Database\BaseConnection.php(652): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (\'asdasds\', \'1111111111111111\', \'sdfsfsf\', \'1111111111111111\', \'sdfsfsdf\', \'1\', \'2025-07-11 09:39:48\', \'2025-07-11 09:39:48\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2345): CodeIgniter\Database\BaseConnection->query('INSERT INTO `iden_info` (`aadhar`, `panno`, `voterid`, `passport`, `other`, `emp_id`, `created_at`, `updated_at`) VALUES (:aadhar:, :panno:, :voterid:, :passport:, :other:, :emp_id:, :created_at:, :updated_at:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(843): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(800): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\RmsController.php(130): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\RmsController->identification_details_save()
10 SYSTEMPATH\CodeIgniter.php(507): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\RmsController))
11 SYSTEMPATH\CodeIgniter.php(354): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(334): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(59): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
DEBUG - 2025-07-11 10:01:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 11:24:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
DEBUG - 2025-07-11 11:24:33 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
